OxfordBodleian/Juel-Jensen/BDLaethf34.xml

```diff
@@ -36,23 +40,484 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        </msItem>
+                        <msItem xml:id="ms_i1.4">
+                            <locus from="8rb" to="10ra"/>
+                            <title type="complete" ref="LIT6282Nesebbeho"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <locus from="8rb" to="8va"/>
+                                <hi rend="rubric">ንሴብሖ፡ ለእግዚአብ</hi>ሔር፡ ለዘበስብሐት፡ ቅዱሳን፡ ይሴባሕ፡ ዘኪያሁ፡ ይሴብሑ፡ ማኅበረ፡ መላእክት፡ ፍሡሓን። ወሎቱ፡ ይትቀነዩ፡ ነፍሳተ፡ ጻድቃን። 
+                                ወሎቱ፡ ትሰግድ፡ ቅድስት፡ ቤተ፡ ክርስቲያ<pb n="8v"/><cb n="a"/>ን፡ እንዘ፡ ትብል።
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                <locus from="9vb" to="10ra"/>
+                                <hi rend="rubric">ገነይነ፡</hi> <del rend="expunctuated">ገነ፡</del> <subst><del rend="erasure" unit="chars" quantity="4"/><add place="overstrike">ለኪ፡ ገነ፡</add></subst>
+                                ለኪ፡ ኦወለተ፡ ዳዊት፡ ገነት፡ ዕፁት፡ አዘቅት፡ ኅትምት፡ ወለቶሙ፡ አንቲ፡ ለነቢ<pb n="10r"/><cb n="a"/>ያት። <hi rend="rubric">ገነይነ፡</hi> ገነ፡ ለ<supplied reason="omitted">ኪ</supplied>፡ 
+                                ገነ፡ ለኪ፡ ኦወለተ፡ ዳዊት። ሐዳስ፡ ቀመር፡ ታቦተ፡ ምሥጢር፡ ኰስኰሰ፡ ወርቅ፡ ዘበሩር። ገነ፡ ለኪ፡ ገ፡ ለ፡
+                            </explicit>
+                        </msItem>
+                            <msItem xml:id="ms_i1.5">
+                            <locus from="10ra" to="11rb"/>
+                                <title type="complete" ref="LIT6283SaaliLana"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez">
+                                <locus from="10ra" to="10rb"/>
+                                <hi rend="rubric">ሰአሊ፡ ለነ፡ ማርያም፡</hi> እምነ፡ ወእሙ፡ ለእግዚእነ፡ ሞገስነ፡ አንቲ፡ ወክብርነ። ፫ ጊዜ፡ በል። <hi rend="rubric">ሰአሉ፡ ለነ፡</hi> ነቢያት፡ ስኩራነ፡
+                                መንፈስ፡ ምሉአነ፡ ሞገስ፡ አቅርን<subst><del rend="erasure" unit="chars" quantity="2"/><add place="overstrike">ቲሁ፡</add></subst> ለሥሉስ፡ <cb n="b"/> 
+                                <hi rend="rubric">ሰአሉ፡</hi> ለነ፡ አስ፡ ለነ፡ ሰአ፡ ለነ፡
+                            </incipit>
+                            <explicit xml:lang="gez">
+                                <locus target="#11rb"/>
+                                <hi rend="rubric">ሰአሊ፡ ለነ<supplied reason="omitted">፡</supplied></hi> አስ፡ ለ፡ ሰአሊ፡ ለነ፡ ቅድስት፡ ቤተ፡ ክርስቲያን፡ መንፈሳዊት፡ ዓውድ፡ መዝገበ፡ ሥርናይ፡
+                                እንበለ፡ ክር<subst><del rend="erasure" unit="chars" quantity="1"/><add place="overstrike">ዳ</add></subst>ድ፡ ምስአል፡ ወምስጋድ፡ ምጥባሐ፡ ሥጋሁ፡ ለወልድ።
+                                ኵሉ፡ ዘኢያፈቅሮ፡ በል፡
+                            </explicit>
+                        </msItem>
+                            <msItem xml:id="ms_i1.6">
+                            <locus from="11rb" to="13ra"/>
+                            <title><ref type="work" corresp="LIT3522Epistle.4.15-5.10">1 Thessalonians 4,15–5,10</ref></title>
```

Comment:
This and other biblical refs will probably need abbreviating, but we can tackle this at the editorial stage. :)